backstop/tests/breadcrumb.html

```diff
@@ -0,0 +1,163 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html id="theme">
+  <head>
+    <title>Breadcrumb | iTwinUI</title>
+    <link
+      rel="icon"
+      href="../assets/favicon.ico"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <style>
+      .iui-input {
+        width: 450px;
+      }
+    </style>
+    <script src="../assets/theme.js"></script>
+  </head>
+  <body class="iui-body">
+    <theme-button></theme-button>
+    <h1>Breadcrumb</h1>
+    <hr />
+
+    <section id="demo-default">
+
+      <nav
+        class="iui-breadcrumb"
+        aria-label="Breadcrumb"
+      >
+        <button class="iui-button iui-borderless iui-dropdown">
+          <!-- Use folder.svg from @itwin/itwinui-icons package. -->
+          <svg
+            class="iui-icon"
+            aria-hidden="true"
+            viewBox="0 0 16 16"
+          >
+            <path d="M1,13a1,1,0,0,0,1,1H14a1,1,0,0,0,1-1V5a1,1,0,0,0-1-1H7V3A1,1,0,0,0,6,2H2A1,1,0,0,0,1,3Z" />
+          </svg>
+          <svg
+            class="iui-icon"
+            aria-hidden="true"
+            viewBox="0 0 16 16"
+          >
+            <path
+              d="m4.8067 6h6.3953a.27961.27961 0 0 1 .24043.44321l-3.1736 3.45707a.33969.33969 0 0 1 -.48085 0l-3.2217-3.45707a.26909.26909 0 0 1 .24042-.44321z"
+            />
+          </svg>
+        </button>
+        <ol>
+          <li
+            itemscope
+            itemtype="https://schema.org/breadcrumb"
+          >
+            <a
+              href="#"
+              itemprop="url"
+            >
+              <span itemprop="title">Root</span>
+            </a>
```

Comment:
Curious why go with `a` and not `button`. These are not going to be hyperlinks if they are changing content inside the page (e.g. folder navigation in a table).

Reply:
Just pushed an update that uses both `<a>` and `<button>`.

Reply:
I think we need to provide a `BreadcrumbItem` component in react that can handle both links and buttons and automatically apply correct styling. Using a button should apply `aria-current=location` on the current item and using a link should apply `aria-current=page`.

Reply:
@FlyersPh9 do we have a standard for page breadcrumbs/navigation? Connect has this one:
![image](https://user-images.githubusercontent.com/36186912/119302176-5d230c80-bc6c-11eb-9a63-984623ced49a.png)
In other places we were using sidenav and title with back button:
![image](https://user-images.githubusercontent.com/36186912/119302287-893e8d80-bc6c-11eb-914e-e1e5a60f8d2b.png)

Reply:
We do have a [folder navigation standard](https://ux.bentley.com/itwin/patterns/folder-navigation.html), and [the header components](https://ux.bentley.com/itwin/patterns/header.html) (new & old designs) have breadcrumbs within them.  The demo HTML does support both `<a>` and `<button>` because I'm not sure what the users might need.

Reply:
The problem is that you can't put these breadcrumbs on a blue background, and you can't use our header either because that's different.

Reply:
The [Portal Header](https://ux.bentley.com/itwin/patterns/header.html#portal) & [Secondary Header](https://ux.bentley.com/itwin/patterns/header.html#secondary) are the older designs intended for use on CONNECT.  All the iTwin apps going forward should be using the [consolidated header](https://ux.bentley.com/itwin/patterns/header.html#consolidated).

If they felt like updating CONNECT to use iTwinUI's breadcrumb it should be a pretty simple text color override to make it work.